UNLV-CS472-672/2024-S-GROUP5-Munch
# Homepage outline

## Description / Changes Made

- Created the format for the home page
- Added buttons for like, comment, and share
- Added avatar
- Created background image
- Added heart.png to the assets folder (like button asset)

## How to Test

1. Navigate to home bar
2. Press Home
3. Check if the home page successfully loads

## Checklist
